charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,1580 @@ spec:
+                                    enabled:
+                                      type: boolean
+                                    key:
+                                      type: string
+                                    secretName:
+                                      type: string
+                                useHadoopConfig:
+                                  type: boolean
+                        environment:
+                          type: string
+                        maxQueryLength:
+                          type: string
+                        nodeSchedulerIncludeCoordinator:
+                          type: boolean
+                        tls:
+                          type: object
+                          properties:
+                            caCertificate:
+                              type: string
+                            certificate:
+                              type: string
+                            createSecret:
+                              type: boolean
+                            enabled:
+                              type: boolean
+                            key:
+                              type: string
+                            secretName:
+                              type: string
+                    coordinator:
+                      type: object
+                      properties:
+                        affinity:
+                          type: object
+                          required:
+                          - podAntiAffinity
+                          properties:
+                            podAntiAffinity:
+                              type: object
+                              required:
+                              - requiredDuringSchedulingIgnoredDuringExecution
+                              properties:
+                                requiredDuringSchedulingIgnoredDuringExecution:
+                                  type: array
+                                  items:
+                                    type: object
+                                    required:
+                                    - labelSelector
+                                    - topologyKey
+                                    properties:
+                                      labelSelector:
+                                        type: object
+                                        required:
+                                        - matchExpressions
+                                        properties:
+                                          matchExpressions:
+                                            type: array
+                                            items:
+                                              type: object
+                                              required:
+                                              - key
+                                              - operator
+                                              - values
```

Comment:
I don't think `value` is required https://github.com/coreos/prometheus-operator/blob/master/example/prometheus-operator-crd/prometheus.crd.yaml#L533-L534